src/core/request.js

```diff
@@ -70,17 +71,25 @@ Request.prototype.request = function(servers, callback_) {
     }
   }
 
-  function onReconnect() {
-    doRequest();
-  }
+  const timeout = setTimeout(() => {
+    if (typeof callback === 'function') {
+      callback(new RippleError('tejTimeout'));
```

Comment:
`tej` is supposed to refer to a local transaction result, not a generic result for all types of requests, but...